app/controllers/v1/bank_statements_controller.rb

```diff
@@ -1,8 +1,12 @@
 module V1
   class BankStatementsController < ApiController
     include MalwareScanning
-    ATTACHMENT_TYPE = "bank_statement_evidence".freeze
-    ATTACHMENT_TYPE_CAPTURE = /^#{ATTACHMENT_TYPE}_(\d+)$/
+
+    def initialize
+      super
+      @attachment_type = "bank_statement_evidence"
```

Comment:
as with other comment could use an attribute reader

```
attr_reader :attachment_type, :attachment_type_capture
```
and use transparently and/or explicitly below